gulpfile.js

```diff
@@ -5,6 +5,14 @@ require('es6-promise').polyfill();
 var gulp = require('gulp');
 var wrench = require('wrench');
 
+/*
+  concatenate all *.js / *.coffee files in the 'gulp' folder
+ */
+wrench.readdirSyncRecursive('./gulp').filter(function(file) {
+  return (/\.(js|coffee)$/i).test(file);
+}).map(function(file) {
+  require('./gulp/' + file);
+});
```

Comment:
Yeah it is.. you can't run the gulp test or build with out it, didn't realise until I set it up.